dev-env/windows/manifests/msys2.json

```diff
@@ -2,21 +2,21 @@
     "homepage": "http://msys2.github.io",
     "version": "20200903",
     "url": [
-        "https://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20200903.tar.xz",
+        "https://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20210228.tar.xz",
         "https://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-jq-1.6-4-any.pkg.tar.zst#/jq.msys2",
         "https://repo.msys2.org/msys/x86_64/gnu-netcat-0.7.1-1-x86_64.pkg.tar.xz#/netcat.msys2",
         "https://repo.msys2.org/msys/x86_64/patch-2.7.6-1-x86_64.pkg.tar.xz#/patch.msys2",
-        "https://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-openssl-1.1.1.g-1-any.pkg.tar.xz#/openssl.msys2",
-        "https://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-python-3.8.2-2-any.pkg.tar.xz#/python.msys2",
+        "https://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-openssl-1.1.1.j-1-any.pkg.tar.zst#/openssl.msys2",
+        "https://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-python-3.8.8-2-any.pkg.tar.zst#/python.msys2",
```

Comment:
Not entirely sure if those are needed but I see no reason to stick to outdated openssl or python versions.

Reply:
Any reason to have fragments in those URLs? IIRC they are not supposed to matter to the server (per HTTP spec), and I can't imagine what scoop might want to do with them.

Reply:
scoop uses them as the file name to download to.

Reply:
Thanks. That is a little bit horrifying...

Reply:
I agree, I tried to delete it at some point which is how I learned this :slightly_smiling_face: